File: aurora-scala/src/main/scala/net/gree/aurora/scala/domain/clustergroup/ClusterGroupIdImpl.scala
package net.gree.aurora.scala.domain.clustergroup

import net.gree.aurora.domain.clustergroup.{ClusterGroupId => JClusterGroupId}

private[domain]
class ClusterGroupIdImpl(val underlying: JClusterGroupId) extends ClusterGroupId {

  def value: String = underlying.getValue

}

File: aurora-core/src/main/java/net/gree/aurora/domain/clustergroup/ClusterGroupIdImpl.java
package net.gree.aurora.domain.clustergroup;

import org.sisioh.dddbase.model.impl.AbstractIdentity;


final class ClusterGroupIdImpl extends AbstractIdentity<String> implements ClusterGroupId {

    /**
     * インスタンスを生成する。
     *
     * @param value 識別子の値
     */
    public ClusterGroupIdImpl(String value) {
        super(value);
    }

}
